src/components/tz.rs

```diff
@@ -109,9 +120,79 @@ impl TimeZone {
         }
     }
 
+    pub fn get_epoch_nanoseconds_for(
+        &self,
+        iso: IsoDateTime,
+        disambiguation: Disambiguation,
+        provider: &mut impl TzProvider,
```

Comment:
It doesn't need to be mutable anymore, since we changed the trait methods to be `&self`. Also applies to many other functions.

Reply:
Shoot, I meant to change that, but forgot.